modules/ROOT/pages/questions/question-15.adoc

```diff
@@ -0,0 +1,9 @@
+15. According to https://ephocks.atlassian.net/browse/DOC-1533[DOC-1533]:
```

Comment:
@lnewson TINY-8382

Reply:
See the answer to Q13, but the gotcha is you need to make sure the schema is configured to allow the elements/attributes used in the media elements now as they can no longer be treated as "special" (where the contents is ignored) since we're using the native [DOMParser API](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser) behind the scenes now.